applications/DEMApplication/tests/test_DEM_3D_inlet.py

```diff
@@ -0,0 +1,58 @@
+import os
+import KratosMultiphysics
+from KratosMultiphysics import Logger
+Logger.GetDefaultOutput().SetSeverity(Logger.Severity.WARNING)
+import KratosMultiphysics.DEMApplication as DEM
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.DEMApplication.DEM_analysis_stage
+
+import auxiliary_functions_for_tests
+
+this_working_dir_backup = os.getcwd()
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+
+class DEM3D_InletTestSolution(KratosMultiphysics.DEMApplication.DEM_analysis_stage.DEMAnalysisStage, KratosUnittest.TestCase):
+
+    @classmethod
+    def GetMainPath(self):
+        return os.path.join(os.path.dirname(os.path.realpath(__file__)), "DEM3D_inlet_tests_files")
+
+    def GetProblemNameWithPath(self):
+        return os.path.join(self.main_path, self.DEM_parameters["problem_name"].GetString())
+
+    def FinalizeSolutionStep(self):
+        super().FinalizeSolutionStep()
+        tolerance = 1.001
+        for node in self.spheres_model_part.Nodes:
+            node_vel = node.GetSolutionStepValue(KratosMultiphysics.VELOCITY_Y)
+            node_force = node.GetSolutionStepValue(KratosMultiphysics.TOTAL_FORCES_Y)
+            if node.Id == 6:
+                if self.time >= 1.15:
+                    Logger.PrintInfo(node_vel)
+                    Logger.PrintInfo(node_force)
+                    self.assertAlmostEqual(node_vel, 0.380489240, delta=tolerance)
+                    self.assertAlmostEqual(node_force, -120983.1002, delta=tolerance)
+
+    def Finalize(self):
+        #self.procedures.RemoveFoldersWithResults(str(self.main_path), str(self.problem_name), '')
```

Comment:
Was this test for this PR? And... don't you want to delete these files?

Reply:
Forgot to reactivate it after checking the post files. me bad